evalml/data_checks/target_leakage_data_check.py

```diff
@@ -21,12 +21,12 @@ class TargetLeakageDataCheck(DataCheck):
 
     Args:
         pct_corr_threshold (float): The correlation threshold to be considered leakage. Defaults to 0.95.
-        method (string): The method to determine correlation. Use 'max' for the maximum correlation, or for specific correlation metrics, use their name (ie 'mutual_info' for mutual information, 'pearson' for Pearson correlation, etc).
+        method (string): The method to determine correlation. Use 'all' or 'max' for the maximum correlation, or for specific correlation metrics, use their name (ie 'mutual_info' for mutual information, 'pearson' for Pearson correlation, etc).
```

Comment:
Can we have a test to make sure that the results of setting `all` and `max` are the same?